src/webpack/common.tsx

```diff
@@ -82,6 +86,10 @@ export const Toasts = {
     }
 };
 
+export const UserUtils = {
+    getUser: lazyWebpack(filters.byCode(".USER(", "getUser")),
```

Comment:
maybe rename to fetchUser or userGet? I know discord themselves name this getUser but that name is more self explanatory. also type would be cool 
```suggestion
    getUser: lazyWebpack(filters.byCode(".USER(", "getUser")) as (id: string) => Promise<User>,
```